src/deploy-cromwell-on-azure/scripts/mount_containers.sh

```diff
@@ -128,4 +128,4 @@ docker_compose_overrides="version: \"3.6\"\nservices:\n  cromwell:\n    volumes:
 echo -e "$docker_compose_overrides" > "docker-compose.override.yml"
 
 # Execute fstab
-sudo mount -av -t fuse
\ No newline at end of file
+echo "$(sudo mount -av -t fuse)"
```

Comment:
The output of startup.sh, including the output of mount_container.sh, is already logged to /cromwellazure/startup.log.

Reply:
I know the output is logged, but this prevents the command from exiting the startup.sh script prematurely. Tested multiple times to confirm.

Reply:
@jacorbello Wrapping the failing command in echo $(...) will allow the execution to continue, but the system will still remain in broken state. In this case, the Cromwell container will fail to start because it can't find the configuration file, since it was not mounted. To address this, we'll add code to capture the errors in startup.sh and add additional step in Deployer.cs to assert that startup was successful before moving on to the "waiting for docker containers" step.